api/controllers/v2/jobs.ts

```diff
@@ -58,14 +58,13 @@ export const HandleJobs = async (event: SQSEvent): Promise<void> => {
   await Promise.all(
     messages.map(async (message: SQSRecord) => {
       const consoleLogger = new ConsoleLogger();
+      console.log('SQS RECORD: ', message);
       const body = JSON.parse(message.body);
       const jobId = body['jobId'];
       const jobStatus = body['jobStatus'];
+
       try {
         switch (jobStatus) {
-          case JobStatus[JobStatus.inQueue]:
```

Comment:
Since there really isn't an in queue period, I decided to remove this. Basically, you get sent the identical message twice in a row which seems jarring when they come one right after another.